package/MDAnalysis/core/universe.py

```diff
@@ -0,0 +1,683 @@
+    :class:`~Universe.select_atoms` for details)::
+
+       u.select_atoms(...)
+
+    Parameters
+    ----------
+    topology : filename or Topology object
+        A CHARMM/XPLOR PSF topology file, PDB file or Gromacs GRO file; used to
+        define the list of atoms. If the file includes bond information,
+        partial charges, atom masses, ... then these data will be available to
+        MDAnalysis. A "structure" file (PSF, PDB or GRO, in the sense of a
+        topology) is always required. Alternatively, an existing
+        :class:`MDAnalysis.core.topology.Topology` instance may also be given.
+    topology_format
+        Provide the file format of the topology file; ``None`` guesses it from
+        the file extension [``None``] Can also pass a subclass of
+        :class:`MDAnalysis.topology.base.TopologyReader` to define a custom
+        reader to be used on the topology file.
+    format
+        Provide the file format of the coordinate or trajectory file; ``None``
+        guesses it from the file extension. Note that this keyword has no
+        effect if a list of file names is supplied because the "chained" reader
+        has to guess the file format for each individual list member.
+        [``None``] Can also pass a subclass of
+        :class:`MDAnalysis.coordinates.base.Reader` to define a custom reader
+        to be used on the trajectory file.
+    guess_bonds
+        Once Universe has been loaded, attempt to guess the connectivity
+        between atoms.  This will populate the .bonds .angles and .dihedrals
+        attributes of the Universe.
+    vdwradii
+        For use with *guess_bonds*. Supply a dict giving a vdwradii for each
+        atom type which are used in guessing bonds.
+    is_anchor
+        When unpickling instances of
+        :class:`MDAnalysis.core.AtomGroup.AtomGroup` existing Universes are
+        searched for one where to anchor those atoms. Set to ``False`` to
+        prevent this Universe from being considered. [``True``]
+    anchor_name
+        Setting to other than ``None`` will cause
+        :class:`MDAnalysis.core.AtomGroup.AtomGroup` instances pickled from the
+        Universe to only unpickle if a compatible Universe with matching
+        *anchor_name* is found. *is_anchor* will be ignored in this case but
+        will still be honored when unpickling
+        :class:`MDAnalysis.core.AtomGroup.AtomGroup` instances pickled with
+        *anchor_name*==``None``. [``None``]
+
+    Attributes
+    ----------
+    trajectory
+        currently loaded trajectory reader;
+    dimensions
+        current system dimensions (simulation unit cell, if set in the
+        trajectory)
+    atoms, residues, segments
+        master Groups for each topology level
+    bonds, angles, dihedrals
+        master ConnectivityGroups for each connectivity type
+
+    """
+
+    def __init__(self, *args, **kwargs):
+        from ..topology.core import get_parser_for
```

Comment:
These imports are in the same place as on the develop branch, which is most likely due to circular imports.  Not sure if this is fixed in 363, we did split AtomGroup and Universe which might help..